modules/tinymce/src/plugins/template/test/ts/browser/DialogGetPreviewContentTest.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-len */
+import { Log, Pipeline, Step } from '@ephox/agar';
+import { UnitTest, Assert } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import TemplatePlugin from 'tinymce/plugins/template/Plugin';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import { getPreviewContent } from 'tinymce/plugins/template/ui/Dialog';
+import Env from 'tinymce/core/api/Env';
+
+const metaKey = Env.mac ? 'e.metaKey' : 'e.ctrlKey && !e.altKey';
+
+const noCorsNoStyle = '<!DOCTYPE html><html><head><base href="http://localhost:8000/"><link type="text/css" rel="stylesheet" href="http://localhost:8000/project/tinymce/js/tinymce/skins/ui/oxide/content.min.css"><link type="text/css" rel="stylesheet" href="http://localhost:8000/project/tinymce/js/tinymce/skins/content/default/content.css"><script>document.addEventListener && document.addEventListener("click", function(e) {for (var elm = e.target; elm; elm = elm.parentNode) {if (elm.nodeName === "A" && !(' + metaKey + ')) {e.preventDefault();}}}, false);</script> </head><body class=""></body></html>';
+
+const CorsNoStyle = '<!DOCTYPE html><html><head><base href=\"http://localhost:8000/\"><link type=\"text/css\" rel=\"stylesheet\" href=\"http://localhost:8000/project/tinymce/js/tinymce/skins/ui/oxide/content.min.css\" crossorigin=\"anonymous\"><link type=\"text/css\" rel=\"stylesheet\" href=\"http://localhost:8000/project/tinymce/js/tinymce/skins/content/default/content.css\" crossorigin=\"anonymous\"><script>document.addEventListener && document.addEventListener(\"click\", function(e) {for (var elm = e.target; elm; elm = elm.parentNode) {if (elm.nodeName === \"A\" && !(' + metaKey + ')) {e.preventDefault();}}}, false);</script> </head><body class=\"\"></body></html>';
```

Comment:
Maybe lower case the first letter of these to be consistent also these could be template strings that would allow you to have multi-line strings and also not need to escape any " or ' letters.

Reply:
oh, I missed this, so nice catch Spocke! Yeah, these should be lowercase as it's a variable name.

- [ ] Lowercase variable name and consider converting to template strings